pkg/kritis/violation/strategy.go

```diff
@@ -63,8 +64,8 @@ func (a *AnnotationStrategy) HandleViolation(image string, pod *v1.Pod, violatio
 		}
 		annotationValue += string(v.Reason) + "\n"
 	}
-	labels := map[string]string{constants.InvalidImageSecPolicyLabel: labelValue}
-	annotations := map[string]string{constants.InvalidImageSecPolicyLabel: annotationValue}
+	labels := map[string]string{kritis.GroupName: labelValue}
```

Comment:
The label name should be "{grpname}.{LabelName}".format(kritis.GroupName, constatants.InvalidImageSecPolicy
Label)

Reply:
done